jsapp/js/components/textFormatter/textFormatter.component.tsx

```diff
@@ -0,0 +1,56 @@
+import type {ReactNode} from 'react';
+
+const regExpMatchParts = /[^*[\]]+|(\*\*[^*]*(?:\*[^*]+)*\*\*)|(\*[^*]+\*)|(\[.+?\]\(.+?\)({:.+})?)/g;
```

Comment:
Wouldn't some existing library satisfy our needs here? Like https://www.npmjs.com/package/markdown-to-jsx or https://www.npmjs.com/package/react-markdown or https://www.npmjs.com/package/marked ?

Reply:
I've used `react-markdown` before and suggest it. I like most the ability to override all components with custom implementation, that's useful especially with Link-s.